Sources/SpeziLLMOpenAI/Onboarding/LLMOpenAIModelOnboardingStep.swift

```diff
@@ -36,7 +36,7 @@ public struct LLMOpenAIModelOnboardingStep: View {
                 Picker(String(localized: "OPENAI_MODEL_SELECTION_DESCRIPTION", bundle: .module), selection: $modelSelection) {
                     ForEach(models, id: \.self) { model in
                         Text(model.formattedModelDescription)
-                            .tag(model.formattedModelDescription)
```

Comment:
Great catch @vishnuravi, I ran into the same issue a couple of days ago and fixed it on my LLM fog branch! However, the Fog LLM capabilities will take a couple more days to complete, therefore it would be great to merge the PR!